static/src/javascripts/projects/commercial/modules/dfp/render-advert-label.ts

```diff
@@ -11,8 +11,9 @@ const shouldRenderLabel = (adSlotNode: HTMLElement) =>
 		adSlotNode.classList.contains('ad-slot--fluid') ||
 		adSlotNode.classList.contains('ad-slot--frame') ||
 		adSlotNode.classList.contains('ad-slot--gc') ||
-		// u-h class is set for out-of-page ads i.e. 1x1 and 2x2 ads
+		// u-h and ad-slot-collapse class is set for out-of-page ads i.e. 1x1 and 2x2 ads
 		adSlotNode.classList.contains('u-h') ||
```

Comment:
If this is successful, I think we should forgo using the `u-h` class entirely

Reply:
Yep thats the plan